test/e2e/streaming_test.go

```diff
@@ -166,6 +166,10 @@ func jaegerStreamingDefinition(namespace string, name string) *v1.Jaeger {
 				Options: v1.NewOptions(map[string]interface{}{
 					"kafka.producer.topic":   "jaeger-spans",
 					"kafka.producer.brokers": kafkaClusterURL,
+					// The following 3 flags are not required for this test, but are added to ensure we pass them passed correctly
```

Comment:
nit: "but are added to ensure we pass them passed correctly" - may be 'passed', needs rewording